lnrpc/signrpc/signer_server.go

```diff
@@ -605,6 +606,13 @@ func (s *Server) SignMessage(_ context.Context,
 		return nil, fmt.Errorf("compact format can not be used for " +
 			"Schnorr signatures")
 	}
+	if !in.SchnorrSig && len(in.Tag) > 0 {
+		return nil, fmt.Errorf("tag can only be used when the Schnorr" +
+			" signature option is set")
```

Comment:
nit: the space between `Schnorr` and `signature` should be on the same line as `Schnorr`